db/migrate/20210104075025_add_active_to_trains.rb

```diff
@@ -0,0 +1,5 @@
+class AddActiveToTrains < ActiveRecord::Migration[6.0]
+  def change
+    add_column :trains, :active, :boolean
```

Comment:
since these migrations havent actually been committed to anywhere just fix the original migration and redo the schema, otherwise we have extra migrations that dont matter